tests/unittests/MLTest.cpp

```diff
@@ -908,7 +908,7 @@ TEST_P(InterpreterAndCPU, convNetForImageRecognition) {
   auto *TANH = F->createTanh("tanh", CV);
   auto *FCL = F->createFullyConnected("fc", TANH, 2);
   auto *SM = F->createSoftMax("sm", FCL, ex);
-  auto *result = F->createSave("ret", SM);
+  auto *resultVar = F->createSave("ret", SM)->getVariable();
```

Comment:
Can you do it in two lines like this?
```c++
auto *SN = F->createSave("ret", SM);
auto *resultVar = SN->getVariable();
```
Otherwise this lines does a bit too much for one line ;-)